exonum-java-binding-cryptocurrency-demo/src/test/java/com/exonum/binding/cryptocurrency/transactions/CreateWalletTransactionUtils.java

```diff
@@ -0,0 +1,45 @@
+package com.exonum.binding.cryptocurrency.transactions;
+
+import static com.exonum.binding.cryptocurrency.CryptocurrencyServiceImpl.CRYPTO_FUNCTION;
+import static com.exonum.binding.cryptocurrency.transactions.CryptocurrencyTransactionTemplate.newCryptocurrencyTransactionBuilder;
+
+import com.exonum.binding.crypto.KeyPair;
+import com.exonum.binding.crypto.PublicKey;
+import com.exonum.binding.messages.BinaryMessage;
+import com.exonum.binding.messages.Message;
+import com.google.protobuf.ByteString;
+
+public class CreateWalletTransactionUtils {
```

Comment:
I'd use the lowest needed access level for these utility classes and their methods as they are used in particular classes only (private or package-private); also, since they are utility classes, I'd forbid instantiation. This will make code suggestions as precise as they used to be and will help avoid confusion about this class purpose (as internal utility class).

Reply:
Ok.